packages/Caching/Contract/Rector/ZeroCacheRectorInterface.php

```diff
@@ -1,13 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Rector\Caching\Contract\Rector;
-
-/**
- * Rectors implementing this interface require to run with --clear-cache, so full application is analysed.
- * Such rules can be remove unused public method, remove unused class etc. They need full app to decide correctly.
- */
-interface ZeroCacheRectorInterface
-{
-}
```

Comment:
@mssimi Hey, we've finally cleanup the broken dead-code rules :) thanks for the interface, it's easier to find them now